README.md

```diff
@@ -0,0 +1,82 @@
+<a name="readme-top"></a>
+
```

Comment:
- [ ] Awesome adding README file, but for it to be more informative and engaging; kindly ensure that all the sections of [Microverse template] are customized and filled out. The only optional section is FAQ and you can indicate "Not yet available" for the live demo section. Kindly don't forget to update your table of contents and ensure that no link(s) is/are broken. ℹ️

Reply:
Hello @OyePriscilla, I have made the changes. Should I create a new pull request?

Reply:
When I renamed the branch it created a new branch @OyePriscilla

Reply:
@rmiyoyo That's awesome, keep doing the good work!